app/scripts/controllers/reviewRegistration.js

```diff
@@ -121,14 +118,28 @@ angular
       });
     }
 
+    function getRegistrantType(typeId) {
+      return _.find(conference.registrantTypes, {
+        id: typeId,
+      });
+    }
+
+    function primaryRegType(currentRegistration) {
+      let primaryRegistrant = _.find(currentRegistration.registrants, {
+        id: currentRegistration.primaryRegistrantId,
+      });
+      return getRegistrantType(primaryRegistrant.registrantTypeId);
+    }
+
     // Navigate to the correct page after completing a registration
-    function navigateToPostRegistrationPage() {
-      if ($scope.regType.registrationCompleteRedirect) {
-        $window.location.href = $scope.regType.registrationCompleteRedirect;
+    $scope.navigateToPostRegistrationPage = function() {
+      let regType = primaryRegType(currentRegistration);
+      if (regType.registrationCompleteRedirect) {
```

Comment:
Could probably do ternary here as well:
```suggestion
regType.registrationCompleteRedirect
        ? ($window.location.href = regType.registrationCompleteRedirect)
        : $route.reload();
```

Reply:
done